docs/projects/import-export/_posts/2019-10-01-javascript.md

```diff
@@ -0,0 +1,57 @@
+---
+title: importing and exporting a variable in JavaScript  
+layout: default  
+last-modified: 2019-10-01  
+featured-image:  
+tags: [javascript, import, export]  
+authors:  
+  - ray6464
```

Comment:
Have you added yourself to [the authors file](https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/_data/authors.yml)?

Reply:
Should I add myself to the file on the following link?
https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/_data/authors.yml

Reply:
Added myself to author's file